version/version.go

```diff
@@ -0,0 +1,26 @@
+package version
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var (
+	// Version is Clamp's version.
+	Version = "unknown version"
+	// Commit is the hash of the commit used to build Clamp.
+	Commit = "unknown commit"
+	// Date is when Clamp was built.
+	Date = "unknown build date"
```

Comment:
```suggestion
	Version = "dev"
	// Commit is the hash of the commit used to build Clamp.
	Commit = "n/a"
	// Date is when Clamp was built.
	Date = "n/a"
```

Reply:
Fixed. The default values are now back to what they were originally.